src/index.html

```diff
@@ -7,7 +7,56 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </div>
+
+    <div class="stars stars--1">
+      <span class="stars__active"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </div>
+
+    <div class="stars stars--2">
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </div>
+
+    <div class="stars stars--3">
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </div>
+
+    <div class="stars stars--4">
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__star"></span>
+    </div>
+
+    <div class="stars stars--5">
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+      <span class="stars__active"></span>
+    </div>
+
   </body>
```

Comment:
```suggestion
    </div>
  </body>
```
1'st point in checklist